src/page-routes/PageRoutes.js

```diff
@@ -13,6 +12,14 @@ import PageRacialDisparities from "../page-racial-disparities/PageRacialDisparit
 const PagesContainer = styled.main``;
 
 export default function PageRoutes() {
+  const location = useLocation();
+
+  useEffect(() => {
+    // hacky nonsense to make the page scroll to the top after navigation,
```

Comment:
Got to love it when a commit containing the comment "hacky nonsense" makes it into the codebase.

Reply:
psychic leakage